modules/geo-layers/src/tile-layer/tile-layer.js

```diff
@@ -24,6 +24,7 @@ const defaultProps = {
   maxCacheByteSize: null,
   refinementStrategy: STRATEGY_DEFAULT,
   zRange: null,
+  indexing: 'xyz',
```

Comment:
Is `xyz` the actual term? Isn't the TMS index also expressed in x,y,z?
Call it `OSM` (geospatial) or `default` (non-geo)?

Reply:
From my reading, XYZ vs TMS [appear to be the standard terms](https://gist.github.com/tmcw/4954720).

Another example, the [TileJSON spec uses `xyz` and `tms`](https://github.com/mapbox/tilejson-spec/tree/master/2.2.0) (lower case) 

>     // OPTIONAL. Default: "xyz". Either "xyz" or "tms". Influences the y
>     // direction of the tile coordinates.
>     // The global-mercator (aka Spherical Mercator) profile is assumed.
>     "scheme": "xyz",

Reply:
- We don't necessarily have to align with TileJSON, but looks like most of our enums are in lowercase, so this is ok.
- I think `scheme` or `indexingScheme` is a better name than `indexing`.

Reply:
I renamed to `indexingScheme`